widgets/pulseaudio.lua

```diff
@@ -10,15 +10,13 @@ local helpers      = require("lain.helpers")
 local shell        = require("awful.util").shell
 local wibox        = require("wibox")
 local string       = { gmatch = string.gmatch,
-                       match  = string.match,
-                       format = string.format }
-local setmetatable = setmetatable
+                       match  = string.match }
 
 -- PulseAudio volume
 -- lain.widgets.pulseaudio
 
 
-local function worker(args)
```

Comment:
Will revert this to `worker` due to a lack of time at the moment.
But I'll try to create another pull request fixing the multiple instances 'bug' in all widgets as soon as I get to it.